src/Windows/R/Components/Impl/ContainerManager/Implementation/ViewModel/ContainerManagerViewModel.cs

```diff
@@ -295,6 +295,7 @@ private void ContrainersChangedMainThread()
 
             try {
                 await _containers.DeleteAsync(container.Id, cancellationToken);
+                _security.DeleteUserCredentials(BrokerConnectionInfo.GetCredentialAuthority(container.Name));
```

Comment:
Can `DeleteUserCredentials` throw anything?

Reply:
No, it does not.

Reply:
Ok, then let's move it below the `try/catch` block and add `return` after `ShowMessage`.